shacl12-node-expr/index.html

```diff
@@ -1127,8 +1127,8 @@ <h3>List Expressions</h3>
 			shnex:pathValues [ sh:zeroOrMorePath rdfs:subClassOf ] ;
 		] ;
 		# This removes any superclasses that are in the list below
-		shnex:minus <b>( owl:Thing rdfs:Resource )</b> ;
-	] .
+		shnex:remove <b>( owl:Thing rdfs:Resource )</b> ;
+	] .</b>
```

Comment:
The new `</b>` in line 1131 has no effect, as it's preceded by the existing `</b>` in line 1130. I believe the new one should be removed, because the old one follows the closing paren that matches the opening paren which immediately follows the opening `<b>`.
```suggestion
		shnex:remove <b>( owl:Thing rdfs:Resource )</b> ;
	] .
```

Reply:
thanks!